plugins/scanners/fossid/src/test/kotlin/FossIdConfigTest.kt

```diff
@@ -33,153 +33,43 @@ import io.kotest.matchers.shouldBe
 import kotlin.IllegalArgumentException
 
 import org.ossreviewtoolkit.clients.fossid.FossIdRestService
+import org.ossreviewtoolkit.plugins.api.Secret
 
 class FossIdConfigTest : WordSpec({
     "create" should {
```

Comment:
I find it concerning that some big chunks of the test have been removed.
Are you testing somewhere else the default values ?

Reply:
These are all tests related to mapping the raw config to the config class which was previously implemented in the `FossIdConfig` class and is now handled by the plugin API itself. I don't want to test functionality of the API in individual plugins.